### ISSUE TITLE
Error on tsc build

### ISSUE THREAD

**Issue by rkdrnf** [flyover/box2d.ts#45]
Errors are emitted as below on typescript version 3.1.3

```
node_modules/box2d.ts/Box2D/Particle/b2ParticleSystem.ts(102,81): error TS2322: Type '<T>(a: T, b: T) => boolean' is not assignable to type '(a: A, b: B) => boolean'.
  Types of parameters 'b' and 'b' are incompatible.
    Type 'B' is not assignable to type 'A'.
node_modules/box2d.ts/Box2D/Particle/b2ParticleSystem.ts(118,81): error TS2322: Type '<T>(a: T, b: T) => boolean' is not assignable to type '(a: A, b: B) => boolean'.
  Types of parameters 'b' and 'b' are incompatible.
    Type 'B' is not assignable to type 'A'.
```

**Comment by NintendoBen** [comment]
I'm running into this issue as well. Do you know of a workaround?

**Comment by flyover** [comment]
This was fixed in a8fab993e18a2b4ff5e547e0da5928a2fbd521ff.
